src/engraving/dom/engravingitem.cpp

```diff
@@ -2109,6 +2109,43 @@ std::pair<int, float> EngravingItem::barbeat() const
     return std::pair<int, float>(bar + 1, beat + 1 + ticks / static_cast<float>(ticksB));
 }
 
+EngravingItem* EngravingItem::findLinkedInScore(const Score* score) const
+{
+    if (!score || !staff() || !links() || links()->empty()) {
+        return nullptr;
+    }
+
+    Staff* linkedStaffInScore = staff()->findLinkedInScore(score);
+
+    if (!linkedStaffInScore) {
+        return nullptr;
+    }
+
+    for (EngravingObject* linked : *links()) {
+        EngravingItem* linkedItem = toEngravingItem(linked);
+        if (linkedItem->staff() == linkedStaffInScore) {
+            return linkedItem;
+        }
+    }
+
+    return nullptr;
```

Comment:
Wouldn't it be nice to replace this with `findLinkedInStaff(linkedStaffInScore)`, to reduce repetition and to clarify the intent?

Reply:
Oops, that's what I meant to do, I must have forgotten. Done now, thanks